frontend/test/metabase/scenarios/admin/settings/sso.cy.spec.js

```diff
@@ -107,6 +111,9 @@ describe("scenarios > admin > settings > SSO", () => {
 });
 
 function saveSettings() {
-  cy.button("Save Changes").click();
-  cy.findByText("Saved");
+  cy.button("Save changes").click();
+  cy.findByText("Success");
+  // This string linger for far too long in the UI, so we have to wait for it to disappear before we assert on anything else.
```

Comment:
Wouldn't it be faster to then do a page refresh instead of waiting?

Reply:
The test was about **subsequent** tries. Doesn't page refresh actually defeat that?

Reply:
Perhaps wait for the request to finish successfully instead? Anyways, doesn't matter - besides this test is 6 seconds longer.

Reply:
I can test this using API only without relying on the UI elements.
Btw, this is not the equivalent of `cy.wait(6000)`. It doesn't wait for that period of time. If the condition is met before that, the test will continue. I simply had to expand the defaault timeout period of 4000ms because the "Success" button stays on a bit longer.